docs/frontend-system/architecture/60-migrations.md

```diff
@@ -14,7 +14,7 @@ This section provides migration guides for different versions of the frontend sy
 
 This guide is intended for app and plugin authors who have already migrated their code to the new frontend system, and are looking to keep it up to date with the latest changes. These guides do not cover trivial migrations that can be explained in a deprecation message, such as a renamed export.
 
-## 1.31.0
+## 1.31
```

Comment:
This is really why I started this up, wanted to have this align with the previous 😅